apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/PublisherClient.cs

```diff
@@ -120,7 +120,19 @@ public sealed class ClientCreationSettings
                 PublisherServiceApiSettings publisherServiceApiSettings = null,
                 ChannelCredentials credentials = null,
                 string serviceEndpoint = null)
-                : this(clientCount, publisherServiceApiSettings, credentials, serviceEndpoint, EmulatorDetection.None)
+                : this(clientCount, publisherServiceApiSettings, credentials, serviceEndpoint, EmulatorDetection.None, null)
+            {
+            }
+
+            // TODO: Allow null?
+
+            /// <summary>
+            /// Creates a settings object which executes the specified configuration action
+            /// on a client builder after applying other settings.
+            /// </summary>
+            /// <param name="configureAction">The configuration action. Must not be null.</param>
+            public ClientCreationSettings(Action<PublisherServiceApiClientBuilder> configureAction)
```

Comment:
Could we at some point just deprecate `ClientCreationSettings` and rely entirely on the Builder?

Reply:
Actually, I meant, deprecate `ClientCreantionSettings` and have a `PublisherClientBuilder` with a `PublisherServiceApiClientBuilder` property.